src/components/input/math-input.js

```diff
@@ -697,6 +698,59 @@ class MathInput extends React.Component {
         this._updateCursorHandle(true);
     };
 
+    domKeyToMathQuillKey = (key) => {
+        // Numbers
+        if (['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'].includes(key)){
+            return `NUM_${key}`;
+        } 
+
+        // Operators
```

Comment:
I feel like for all the non-integer keys, it would be simpler to have a dictionary and return the value if the key is present. Like

```
const keyMap = {
    "+": Keys.PLUS,
    "-": Keys.MINUS,
    "*": Keys.TIMES,
    "/": Keys.DIVIDE,
...
};

if (key in keyMap) {
    return keyMap[key];
}

return null;
```

or something like that. Thoughts?